app/views/shared/menu/_large_menu.html.haml

```diff
@@ -37,6 +37,12 @@
               = t 'label_about'
 
     %ul.right
+      %li.powered-by
+        %img{src: '/favicon.ico'}
+        %span
+          = t 'powered_by'
+          %a{href: '/'}
+            = t 'the_open_food_network'
```

Comment:
That gives us "OFN UK", which is maybe not what we want. I've just noticed there's another translation key for site title, which has "Open Food Network". Would that be better?